ui/src/components/sermons/OnlinePageTabs.js

```diff
@@ -26,129 +32,222 @@ const OnlinePageTabs = ({ sermonNotes }) => {
     setNoteId(noteId + 1);
   };
 
-  return (
-    <Tabs
-      isFitted
-      h="100%"
-      onChange={(i) => setTab(i)} overflowY={(tab ? 'auto' : 'hidden')}
-    >
-      <TabList orientation="horizontal">
-        <Tab>Sermon Notes</Tab>
-        <Tab>New to HMCC</Tab>
+  const IgnitePage = () => {
+    return (
+      <Tabs
+        isFitted
+        onChange={(i) => setTab(i)} overflowY={(tab ? 'auto' : 'hidden')}
+      >
+        <TabList orientation="horizontal">
+          <Tab 
+          _selected={{
+            color: "#46B2D3",
+            borderBottom: "2px solid",
+            borderBottomColor: "#46B2D3",
+          }}
+          borderBottomColor="black"
+          color="white">
+            About Ignite 2022</Tab>
+          <Tab
```

Comment:
not sure if it's just me, but i have to click the "Schedule" tab 2 times in order for the tab to actually chagne